default_warehouse_from_sale_team/views/stock_view.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<openerp>
+    <data>
+
+       <record id="stock_warehouse_sale_teams" model="ir.ui.view">
+            <field name="name">stock.warehouse.sale.teams</field>
+            <field name="model">stock.picking.type</field>
+            <field name="inherit_id" ref="stock.view_picking_type_form"/>
+            <field name="arch" type="xml">
+                <field name="warehouse_id" position="attributes">
+                    <attribute name="required">1</attribute>
```

Comment:
I am not sure if this validation is necessary here, a picking type is not common to be edited manually.